app/components/Resources/ResourcesTable.js

```diff
@@ -116,26 +119,20 @@ class ResourcesTable extends Component {
 	componentDidMount() {
 		if(this.props.userLocation) {
 			let position = this.props.userLocation;
-			let userPosition = {lat: position.coords.latitude, lng: position.coords.longitude};
+			let userLocation = {lat: position.coords.latitude, lng: position.coords.longitude};
 			this.setState({
-				location: userPosition
+				location: userLocation
 			});
+			this.loadResourcesFromServer(userLocation);
 		} else {
-			this.getLocationGoogle();
-			// this.props.getLocation(position => {
-			// 	let userPosition = {lat: position.coords.latitude, lng: position.coords.longitude};
-			// 	console.log("got it!", userPosition);
-			// 	this.setState({
-			// 		location: userPosition
-			// 	});
-		 //  },
-		 //  error => {
-		 //  	console.error(error);
-		 //  });
+			let self = this;
+			this.getLocationGoogle()
+			.then(loc => {
+				self.loadResourcesFromServer(loc);
+			});
+			
 		}
-
-		this.loadResourcesFromServer();
-
+		this.loadResourcesFromServer({lat: 1, lng: 1});
```

Comment:
Do we want to make this call? Both branches of the `if` above call `loadResourcesFromServer`.